backend/app/infrastructure/routes/allocations.py

```diff
@@ -62,6 +62,10 @@
     "UserAllocationRequest",
     {
         "payload": fields.Nested(allocation_payload),
+        "user_address": fields.String(
```

Comment:
@paulperegud: please change to camelCase, unifying all the fields across the endpoint and most of the stack.
```suggestion
        "userAddress": fields.String(
```

Reply:
[The client is now adjusted and sends camelCased `userAddress`](https://github.com/golemfoundation/octant/pull/112/commits/cb639db6379f6b01d36126f3a688844528bff464).